software/watcharr.yml

```
@@ -0,0 +1,15 @@
+name: Watcharr
+website_url: "https://github.com/sbondCo/Watcharr"
+description: A free and open source content watch list. Add and track all the shows and movies you are watching. Comes with user authentication, modern and clean UI and a very simple setup.
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - Go
```

> https://github.com/sbondCo/Watcharr#set-up

As Docker is the only listed set up method, I would say lets remove `Go` until there is documentation for it.

```suggestion
platforms:
  - Docker
```